static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -427,6 +430,22 @@ const SPECIAL_FIELDS: SpecialFields = {
       </Container>
     ),
   },
+  assignee: {
+    sortField: 'assignee.name',
+    renderFunc: data => {
+      const assignedTo = {
+        type: data['assignee.type'],
+        id: data['assignee.id'],
+        name: data['assignee.name'],
+        email: data['assignee.email'],
+      } as Actor;
```

Comment:
Should we prefer `const assignedTo: Actor = {...}`? I think with `as Actor`, you can add any rando props to the object and the compiler will still treat it as an `Actor`, but with the `: Actor` it'll complain

Reply:
good call! updated!